### ISSUE TITLE
Differences between online demo and offline model

### ISSUE THREAD

**Issue by dangchenyu** [mit-han-lab/temporal-shift-module#92]
Hello, in the online demo, InvertedResidualWithShift returned
return x + self.conv(torch.cat((shift_buffer, x2), dim=1)), x1
but in the offline shift part of MobileNet V2, it returned
return out.view(nt, c, h, w) but without plus initial x, why here is not the same.

**Comment by dangchenyu** [comment]
I have understood it myself. Sorry.
